packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -61,4 +61,59 @@ export default class PaypalCommerceRequestSender {
 
         return res.body;
     }
+
+    async setCountry(countryId: string) {
+        const url = `remote/v1/country-states/${countryId}`;
+        const headers = {
+            'X-API-INTERNAL': INTERNAL_USE_ONLY,
+            'Content-Type': ContentType.Json,
+            ...SDK_VERSION_HEADERS,
+        };
+        const res = await this._requestSender.get<OrderStatus>(url, {headers});
+
+        return res.body;
+    }
+
+    async setShippingQuote(countryId: string, stateId: number, city: string, stateCode: string) {
+        const url = `remote/v1/shipping-quote?city=${city}&country_id=${countryId}&state_id=${stateId}&zip_code=${stateCode}`;
```

Comment:
Can you try to pass `?city=${city}&country_id=${countryId}&state_id=${stateId}&zip_code=${stateCode}` as a `body` object for request sender. I don't know if it works, but it will be great to know

```
const body = {
    city,
    'country_id': countryId,
    'state_id': stateId,
    'zip_code': stateCode,
};
```